src/Locations/Locations.js

```diff
@@ -0,0 +1,43 @@
+
+import React, { Component } from 'react'
+import axios from 'axios'
+import "./locations.css"
+export default class Locations extends Component {
+    constructor(){
+        super()
+        this.state={
+            locations: [],
+            showLocal: false
+        }
+    }
+     
+    getlocations = async()=>{
```

Comment:
Some of the whitespace and spacing is a little bit odd - I'd highly recommend using the vscode extension Prettier to format all of your code! You can set it up to automatically reformat your code each time you save a file.